File: save_image.py
from PIL import Image
import requests
from io import BytesIO
import os

# Create assets directory if it doesn't exist
if not os.path.exists('assets'):
    os.makedirs('assets')

# Save the image
image_path = os.path.join('assets', 'genius.png')
img = Image.new('RGBA', (800, 800), (135, 206, 235, 0))  # Light blue background
img.save(image_path, 'PNG')

print(f'Image saved to {image_path}')

File: save-image.js
const fs = require('fs');
const path = require('path');

// Base64 encoded image data (remplacez ceci par les données de votre image)
const imageData = `
[Votre image en base64]
`;

const imagePath = path.join(__dirname, 'assets', 'genius.png');

// Decode and save the image
const imageBuffer = Buffer.from(imageData, 'base64');
fs.writeFileSync(imagePath, imageBuffer);

console.log('Image saved successfully to:', imagePath);
